sandbox.md

```diff
@@ -0,0 +1,57 @@
+# sandbox keyword
+## motivation
+синтаксический сахар для запуска кода "в песочнице" - не вызывает никаких сайд-эффектов во время исполнения.
```

Comment:
syntax sugar for running the sandboxed code - causes no side effects during execution.